controllers/controllers/workloads/env/vcap_app_builder_test.go

```diff
@@ -63,5 +81,70 @@ var _ = Describe("VCAP_APPLICATION env value builder", func() {
 				Expect(vcapAppValue).To(HaveKeyWithValue("x", HaveKeyWithValue("y", "z")))
 			})
 		})
+
+		When("application routes are provided", func() {
+			var (
+				appDomain *korifiv1alpha1.CFDomain
+				appRoute  *korifiv1alpha1.CFRoute
+				routeUri  string
+			)
+
+			BeforeEach(func() {
+				appDomain = &korifiv1alpha1.CFDomain{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: cfApp.Namespace,
+						Name:      "domain-123-456",
+					},
+					Spec: korifiv1alpha1.CFDomainSpec{
+						Name: "mydomain.platform.com",
+					},
+				}
+				ensureCreate(appDomain)
+
+				appRoute = &korifiv1alpha1.CFRoute{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: cfApp.Namespace,
+						Name:      "vcap-application-my-route",
+					},
+					Spec: korifiv1alpha1.CFRouteSpec{
+						DomainRef: v1.ObjectReference{
+							Name: appDomain.Name,
+						},
+						Destinations: []korifiv1alpha1.Destination{korifiv1alpha1.Destination{
```

Comment:
Golang allows skipping the type declaration of a literal in arrays, i.e. `Destinations` could be a bit shorter:

```
						Destinations: []korifiv1alpha1.Destination{{
							GUID: "destination-123-456",
							AppRef: v1.LocalObjectReference{
								Name: cfApp.Name,
							},
						}}
```